app/code/Magento/Review/Block/Adminhtml/Add.php

```diff
@@ -21,16 +21,16 @@ class Add extends \Magento\Backend\Block\Widget\Form\Container
     protected function _construct()
     {
         parent::_construct();
-
         $this->_blockGroup = 'Magento_Review';
         $this->_controller = 'adminhtml';
         $this->_mode = 'add';
-
         $this->buttonList->update('save', 'label', __('Save Review'));
         $this->buttonList->update('save', 'id', 'save_button');
-
-        $this->buttonList->update('reset', 'id', 'reset_button');
-
+        $this->buttonList->remove('reset');
+        $this->buttonList->add(
+            'reset_button',
+            ['label' => 'Reset', 'onclick'=>"jQuery('#edit_form').trigger('reset')"]
```

Comment:
Hi @arushibansal013. Thanks for collaboration. 
It isn't correct to pass JS code from PHP code. Also, you hardcode HTML selector directly in php code. 
Please move JS code to phtml template and use x-magento-init to inject it.

Reply:
Hi @VladimirZaets,

Can you review it now.